python/src/main/api.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import sqlite3
+
+from flask import Flask
+from flask import jsonify, abort
+
+from python.src.main.db import DB
+from python.src.main.parser import create_person, create_item
+from python.src.main.Person import Person
+
+
+def main():
+    app = Flask(__name__)
+    logging.basicConfig(filename='present_app.log', level=logging.DEBUG)
```

Comment:
Added a logging file, this enables us to capture all of the actions that happen in our app.
Particularly, if things go wrong it allows us to see what went wrong